### PR TITLE
Privacy changes

### PR DESCRIPTION
**Pull request by masterlittle**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [ ] Bug Fix
- [ ] New Query Runner (Data Source)
- [ ] New Alert Destination
- [ ] Other

### CODE REVIEW

File: redash/models/__init__.py [hunk 1]
```diff
@@ -82,6 +81,69 @@ def get(self, query_id):
 scheduled_queries_executions = ScheduledQueriesExecutions()
 
 
+class PolicyTag(db.Model):
```
Comment (avinashdunzo) [inline redash/models/__init__.py:84]: How is migration being run? Is it running automatically?

Comment (masterlittle) [reply redash/models/__init__.py:84]: Will have to run these manually right now.
1. Not very familiar with alembic.
2. These are all new tables so nothing will break.

For any new setup in a db, these will be created automatically.

Comment (avinashdunzo) [reply redash/models/__init__.py:84]: Then checkin a SQL file. will all the create table queries.

Comment (masterlittle) [reply redash/models/__init__.py:84]: Yes. Will do

**Review by avinashdunzo — APPROVED** [review]

**Pull request merged**